app/models/user.rb

```diff
@@ -3,4 +3,14 @@ class User < ApplicationRecord
   has_many :user_bets
   has_many :debts,   class_name: 'Debt', foreign_key: :debtor_id
   has_many :credits, class_name: 'Debt', foreign_key: :creditor_id
+
+  scope :winners, lambda {
+    joins(user_bets: :bet_option)
+      .merge(BetOption.winners)
+  }
+
+  scope :losers, lambda {
+    joins(user_bets: :bet_option)
+      .merge(BetOption.losers)
```

Comment:
Javascript prefers dots at the start of the line 
Ruby usually prefers dot at the end of the previous line (this is an opinion) but reading line by line indicates that there is a continuation on the next line if the dot is at the end, and the code can be copy and pasted into a terminal.

If the dot is at the start of the new line, the code can't be copy pasted directly into a terminal